.travis.yml

```diff
@@ -0,0 +1,27 @@
+sudo: required
+language:
+  # Setting language to objective-c sends us to an OSX VM
+  - objective-c
+before_install:
+  # TavisCI's OSX environment gives us node 0.10.32
+  # Use NVM to update it to 0.12
+  - rm -rf ~/.nvm
+  - git clone https://github.com/creationix/nvm.git ~/.nvm
+  - source ~/.nvm/nvm.sh
+  - nvm install 0.12
+  # Chrome Driver needs Chrome >= 43.0.2357.0
```

Comment:
You might note ChromeDriver version XXX needs this Chrome. This is a moving target.